### ISSUE TITLE
Need for supporting Systemverilog verification keyword highlighting

### ISSUE THREAD

**Issue by ibrahimayaz95** [someone13574/zed-verilog-extension#4]
Hello,
I used the extension for not so long but there is a need for people who are working in the verification area that needs highlighting for .svh (Systemverilog header files) with verification keywords like "class" "extends" and so on.

**Comment by someone13574** [comment]
I'll close this issue once the update is merged into the `zed/extensions` repo.

**Comment by ibrahimayaz95** [comment]
Hello,
Thank you very much for your quick response and action. I hope you have a wonderful week!

**Comment by someone13574** [comment]
Ok, I've opened a PR to the extension repo. Once https://github.com/zed-industries/extensions/pull/1401 is merged it should work as expected.
